### PR TITLE
Insomnia: storage options pages

### PR DESCRIPTION
**Pull request by cloudjumpercat**
- [Main storage options landing page](https://deploy-preview-123--kongdeveloper.netlify.app/insomnia/storage-options/)
  - [scratch pad](https://deploy-preview-123--kongdeveloper.netlify.app/insomnia/scratch-pad/)
  - [git sync](https://deploy-preview-123--kongdeveloper.netlify.app/insomnia/git-sync/)
  - [cloud sync](https://deploy-preview-123--kongdeveloper.netlify.app/insomnia/cloud-sync/)
  - [Version control concept/ref page](https://deploy-preview-123--kongdeveloper.netlify.app/insomnia/version-control/): this one is like API specs, it teaches you a few small tasks you can do for version control, answers some FAQ, etc. But this applies to git sync and cloud sync, so I felt it needed a separate page

Have a few open questions before I finish the version control FAQ and intro paragraph.

### CODE REVIEW

File: app/insomnia/scratch-pad.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  - text: API specs
+    url: /insomnia/api-specs/
+  - text: Write tests for HTTP status codes
+    url: /how-to/write-http-status-tests/
+  - text: Security at Insomnia
+    url: /
+
+---
+
+Local vault (scratch pad) is a storage option that allows all project data to be stored locally on your device. This option is ideal for users who prefer or require their data to remain off the cloud for privacy or security reasons.
```
Comment (lmilan) [inline app/insomnia/scratch-pad.md:26]: Local vault and scratch pad aren't the same thing, I would has separate pages for them

Comment (cloudjumpercat) [reply app/insomnia/scratch-pad.md:26]: Thanks! I didn't realize it

File: app/insomnia/version-control.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+Branch protections, are they available? In cloud? In git?
+
+when should I use git sync vs cloud sync? link to longer form storage about docs
+
+
+## Version control capabilties
+
+The following sections describe how to perform version control actions in Insomnia. 
+
+Before you can manage branches, commits, and merges, click the entity (document, collection, mock server, or global environment) from your workspace.
```
[the remark this answers is not in the document]
Comment (cloudjumpercat) [reply app/insomnia/version-control.md:60]: Ohh, that makes more sense now! I'll change the language there

[the remark this answers is not in the document]
Comment (cloudjumpercat) [reply app/insomnia/version-control.md:60]: Wait, I think the organizations might actually be projects. I think you can have multiple projects in an organization.
Screenshot 2024-10-18 at 4 01 05 PM

**Pull request merged**